src/state/predictions/index.ts

```diff
@@ -66,6 +66,7 @@ const initialState: PredictionsState = {
   ledgers: {},
   claimableStatuses: {},
   leaderboard: {
+    selectedAddress: null,
```

Comment:
This was necessary because it was working on the Leader board only by accident. It spawns from our issue with the modals and props.